sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -61,6 +61,19 @@ grammar SqlBase;
    * When true, the behavior of keywords follows ANSI SQL standard.
    */
   public boolean SQL_standard_keyword_behavior = false;
+
+  /**
+   * Verify whether current token is a valid hint token (which follows '/*' and is '+').
+   * Returns true if the first character is '+'.
+   */
+  public boolean isHint() {
+    int firstChar = _input.LA(1);
```

Comment:
`_input.LA(1)` returns next char, shall we name it `nextChar`?

Reply:
OK